libraries/standard/http/include/http_client.h

```diff
@@ -205,6 +214,11 @@ typedef struct HTTPRequestHeaders
      * and #HTTPClient_InitializeRequestHeaders.
      */
     size_t headersLen;
+
+    /**
+     * @brief Flags to activate other request header configurations.
+     */
+    uint32_t flags;
```

Comment:
Given we won't couple InitialzieRequestHeaders and Addheader let's take this out.